lib/core/src/server/build-dev.js

```diff
@@ -205,7 +209,11 @@ export async function buildDevStandalone(options) {
     console.log(
```

Comment:
Has not changed but I'm wondering why this is a console.log instead of `logger.info`

Reply:
because logger.info adds a green 'info' before what's logged